src/Controller/ForgotPasswordController.php

```diff
@@ -29,8 +30,15 @@ class ForgotPasswordController extends AbstractController
     private const SESSION_TOKEN_KEY = 'forgot_password_token';
     private const SESSION_CAN_CHECK_EMAIL = 'forgot_password_check_email';
 
-    /** @TODO this value should be generated/retrieved from the config... */
-    private const LIFETIME_HOURS = 1;
+    /**
+     * @var ResetPasswordHelperInterface
+     */
+    private $resetPasswordHelper;
+
+    public function __construct(ResetPasswordHelperInterface $resetPasswordHelper)
+    {
+        $this->resetPasswordHelper = $resetPasswordHelper;
+    }
```

Comment:
Because we ALWAYS need this service in this controller, this is a pretty great idea. We'll just need to remember to remove this as an argument to all the methods.